contracts/registries/LicenseRegistry.sol

```diff
@@ -230,12 +233,11 @@ contract LicenseRegistry is ILicenseRegistry, AccessManagedUpgradeable, UUPSUpgr
                 revert Errors.LicenseRegistry__DuplicateLicense(parentIpIds[i], licenseTemplate, licenseTermsIds[i]);
             }
         }
-
         $.licenseTemplates[childIpId] = licenseTemplate;
-        _setExpirationTime(
-            childIpId,
-            ILicenseTemplate(licenseTemplate).getEarlierExpireTime(licenseTermsIds, block.timestamp)
-        );
+        // calculate the earliest expiration time of child IP with both parent IPs and license terms
+        earliestExp = _calculateEarliestExpireTime(earliestExp, licenseTemplate, licenseTermsIds);
+        // default value is 0, means never expired
```

Comment:
suggestion:
default value is 0 which means that the license never expires

Reply:
updated the comments.